src/app/content/highlights/components/EditCard.tsx

```diff
@@ -98,6 +98,9 @@ const EditCard = React.forwardRef<HTMLElement, EditCardProps>((props, ref) => {
     if (element.current) {
       props.onHeightChange(element);
     }
+    if (!props.isActive && !props.data?.annotation) {
+      setPendingAnnotation(defaultAnnotation());
+    }
```

Comment:
@TomWoodward new change here - Malar caught a bug where unsaved note text was being stored in the EditCard's local state even after focus leaves the card. I think it predates all of the recent work.